grip-mode.el

```diff
@@ -98,6 +103,14 @@ option."
 (defvar-local grip--preview-file nil
   "The preview file for grip process.")
 
+(defun grip--browser (url)
+  "Use browser specified by user to load URL.
+Use default browser if nil."
+  (when grip-url-browser
+    (setq-local browse-url-generic-program grip-url-browser
+                browse-url-browser-function 'browse-url-generic))
```

Comment:
Also set `grip-url-args`.

Reply:
will look into it

Reply:
Just use

```elisp
(setq-local browse-url-generic-program grip-url-browser
            browse-url-generic-args grip-url-args
            browse-url-browser-function 'browse-url-generic)
```

Reply:
But what if user wants to pass arguments to default browser. I also though just add it to setq-local, but then it came to mind that maybe user wants to pass args to regular browser as well

Reply:
So the best solution should be use `let`. The local vars won't be override. 
I tried this snippet and it works as expected.

```elisp
(defun grip--browser (url)
  "Use browser specified by user to load URL.
Use default browser if nil."
  (if grip-url-browser
      (let ((browse-url-generic-program grip-url-browser)
            (browse-url-generic-args grip-url-args))
        (ignore browse-url-generic-program)
        (ignore browse-url-generic-args)
        (browse-url-generic url))
    (browse-url url)))
```

Reply:
yes. tested it. works as expected